tests/models/albert/test_modeling_albert.py

```diff
@@ -288,6 +289,12 @@ def setUp(self):
         self.model_tester = AlbertModelTester(self)
         self.config_tester = ConfigTester(self, config_class=AlbertConfig, hidden_size=37)
 
+    @parameterized.expand([("float16",), ("bfloat16",), ("float32",)])
+    @require_torch_sdpa
+    @unittest.skip("Albert requires `head_mask` which is currently not done in this test.")
+    def test_eager_matches_sdpa_inference(self):
+        pass
```

Comment:
On skips like this, on Albert and other models: the test pulls the main input and the attention mask to manipulate them, finally sending them to the model. We could `pop` these items from `inputs_dict`, and then pass `**inputs_dict` to the model (e.g. `model_eager(**prepared_inputs, **inputs_dict)`) -- I think then we wouldn't need to skip tests due to missing inputs 🤗

Reply:
Maybe let's merge and do this in a follow-up PR. It's never worked before.